examples/react/apollo-client-defer/src/App.tsx

```diff
@@ -0,0 +1,56 @@
+import { useQuery } from '@apollo/client';
+
+import { useFragment, graphql, FragmentType, isFragmentReady } from './gql';
+import { SlowAndFastFieldWithDeferQuery } from './gql/graphql';
```

Comment:
We don't want to encourage people to import this type directly. Instead `DocumentType<typeof alphabetQuery>` should be used.

Reply:
Ok, changed that. It wasn't very important here — just for the sake of an example